hr_employee_firstname/tests/test_hr_employee_firstname.py

```diff
@@ -10,24 +10,23 @@
 class TestEmployeeFirstname(TransactionCase):
     def setUp(self):
         super().setUp()
-        self.employee_model = self.env['hr.employee']
```

Comment:
Why this is needed?

Reply:
Why not? I think is a matter of taste...

Reply:
More copypaste of a model name string? + a lookup every time?

Reply:
You like the old style for tests.. I see.
Do you prefer I revert such changes?

Reply:
If to take about personal preferences, I do prefer to reduce copy-paste, yet my personal preference is not a blocking one :) In this particular case I'll just need a second opinion on this, that's all - because truly I'm not sure how it's better for OCA

Reply:
I mean, on one hand, I also did tidy-up things in migration, yet on other hand it's better to keep migration as small as possible 🤷‍♂

Reply:
Reverting the change will speed up the approval process? This PR is pending for a long time, I don't want to wait more.

Reply:
I've asked @pedrobaeza to share his opinion on this, probably will take couple of hours? Yes answering to your question specifically: yes, but I don't want to push for that change

Reply:
If to talk about style preferences, I'd wrote that as `self.HrEmployee` :)))) Yet that is completely irrelevant

Reply:
To cache the model references is a way to not query each time the environment, and to have shorter names for invocating them, so I see this as something correct, but author preference, so I wouldn't argue if I find it or not.

Reply:
@astirpe so if you think that is reasonable, please revert that change, otherwise I'll just merge it as is.

Reply:
Is it okay to merge as it is? Thanks.